tensorflow/tensorflow-engine/src/main/java/ai/djl/tensorflow/engine/TfNDManager.java

```diff
@@ -212,9 +197,9 @@ public NDArray randomUniform(float low, float high, Shape shape, DataType dataTy
         NDArray axes = create(shape.getShape());
         TfOpExecutor opBuilder =
                 opExecutor("RandomUniform").addInput(axes).addParam("dtype", dataType);
-        if (seed != null) {
-            opBuilder.addParam("seed", 1234);
-            opBuilder.addParam("seed2", 1234);
+        if (getEngine().getSeed() != null) {
```

Comment:
1. getSeed() is called multiple times, define a variable instead
2. Should the two seed the same?, should we do something seed + 1?

Reply:
I checked tf source code and change it